### ISSUE TITLE
Not compatible with react 0.14

### ISSUE THREAD

**Issue by harobed** [vkbansal/react-contextmenu#3]
Not compatible with react 0.14 :

```
Uncaught Error: Invariant Violation: ContextMenu.render(): A valid ReactComponent must be returned. You may have returned undefined, an array or some other invalid object
```

**Comment by meikoudras** [comment]
It seems that Flummox (which is used as the Flux implementation here does not support React 0.14 at the moment and it is being deprecated entirely). I wrote a fork of this which uses the Alt Flux implementation (Seemed very similar to Flummox) and merged in the React 0.14 branch available here.

It seems to be working on React 0.14 and you can access it here: 
https://github.com/meikoudras/react-contextmenu

I can make a PR for this but I am not sure if @vkbansal wants to use the Alt library instead of Flummox.

**Comment by vkbansal** [comment]
@meikoudras I initially started out with Alt but later decided to go with Flummox instead.

I know that flummox is deprecated. My plan was to migrate to Redux (As soon as I find some time).
But for time being if Alt works, then I have no problem with it. You are welcome to open a PR.

**Comment by vkbansal** [comment]
Flummox has been updated for 0.14. I've released v0.3 with the update.
